packages/webdriverio/src/types.ts

```diff
@@ -677,6 +677,13 @@ export interface SaveScreenshotOptions {
     }
 }
 
+/**
+ * Additional options outside of the WebDriver spec, exclusively for Webdriverio, only for runtime, and not sent to appium
+ */
+export interface WdioOptions {
```

Comment:
`WdioOptions` seems to general , maybe we can be a bit more specific here?

```suggestion
export interface InputOptions {
```

Reply:
Should we just re-use the `CommandRuntimeOptions` parameter?

Reply:
Now that `CommandRuntimeOptions` is a class, having `InputOptions` as an interface makes it easier to use in the `setValue` and `addValue`, so I would keep both.

Reply:
Resolving since the two are more relevant now, let me know if you disagree!